Update configuration-vocabulary.md and trace-sampling.md (#5409)
apache/skywalking: docs/en/setup/backend/configuration-vocabulary.md

| agent-analyzer | default | Agent Analyzer. | SW_AGENT_ANALYZER | default |
| - | -| sampleRate|Sampling rate for receiving trace. The precision is 1/10000. 10000 means 100% sample in default.|SW_TRACE_SAMPLE_RATE|10000|
| - | - |slowDBAccessThreshold|The slow database access thresholds. Unit ms.|SW_SLOW_DB_THRESHOLD|default:200,mongodb:100|

> | - | - |forceSampleErrorSegment|When sampling mechanism active, this config can open(true) force save some error segment. true is default.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|

Reviewer: ```suggestion
| - | - |forceSampleErrorSegment|When sampling mechanism activated, this config would  make the error status segment sampled, ignoring the sampling rate.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
```

Reviewer: And change the sampling doc when same way. The previous version has some grammar issue.
